pkg/apis/config/trusted_resources.go

```diff
@@ -29,6 +29,7 @@ import (
 // TrustedResources holds the collection of configurations that we attach to contexts.
 // Configmap named with "config-trusted-resources" where cosign pub key path and
 // KMS pub key path can be configured
+// Deprecated.
```

Comment:
Maybe we would like to add 'Use verification policy instead' for pointing users to the alternative.

Reply:
Sure! Thanks for the suggestion!